src/content/admin/resource-manager.mdx

```diff
@@ -108,6 +90,18 @@ If you find that the Resource Manager's recommendations are not suitable for you
 Update the annotation `dev.okteto.com/resource-manager` to `false` to your pod's metadata to prevent automatic adjustments.
 Roll back to previous resource settings manually if needed.
 
+## Disabling the Resource Manager
+If you need to disable the Resource Manager temporarily, you can do so by:
+
+- Using the toggle switch in the Admin UI to disable the Resource Managers automatic adjustments
+<!-- - To uninstall Resource Manager, you can set the `resourceManager` helm setting to `enabled: false` -->
+
 ## Troubleshooting and Support
 ### Metric Server Availability
-The Resource Manager relies on the Kubernetes Metric Server. If the Metric Server is unavailable, the Okteto UI will display an error message. You can troubleshoot the Metric Server or refer to the documentation for further guidance.
\ No newline at end of file
+The Resource Manager relies on the Kubernetes Metric Server. If the Metric Server is unavailable, the Okteto UI will display an error message. You can troubleshoot the Metric Server or refer to the documentation for further guidance.
+
+### Known Limitation: Resource Manager Support for `ReplicaSet` and `CronJob`
+
+The Resource Manager does not natively support optimizing ReplicaSets and CronJobs due to limitations in differentiating between them and their controllers (e.g., Deployments and Jobs). By default, the Resource Manager infers the original controller’s name from the pod’s name (e.g., extracting `${deploymentName}` from a pod named `${deploymentName}-644d69dbc4-fv5g5`).
+
+**Recommendation**: If you need to enable Resource Manager optimizations for ReplicaSets or CronJobs, add the label dev.okteto.com/component-name to your resources. This allows the Resource Manager to use the specified value in the label instead of inferring the controller name.
```

Comment:
```suggestion
**Recommendation**: If you need to enable Resource Manager optimizations for ReplicaSets or CronJobs, add the label `dev.okteto.com/component-name` to your resources. This allows the Resource Manager to use the specified value in the label instead of inferring the controller name.
```